### PR TITLE
Restrict file name length to 64 characters

### PR DESCRIPTION
**Pull request by SYBIOTE**
This pull request restricts the length of file names to 64 characters. Previously, there was no limit on the length of file names, which could lead to issues. This change ensures that file names are within the acceptable range.

### CODE REVIEW

File: packages/common/src/utils/validateFileName.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export function isValidFileName(fileName: string) {
   return (
     !INVALID_FILENAME_WHITESPACE_REGEX.test(fileName) &&
```
Comment (aditya-mitra) [inline packages/common/src/utils/validateFileName.ts:30]: we can restrict the fileName length in the regex in itself. somewhat like `{1,64}`

**Review by aditya-mitra — DISMISSED** [review]

File: packages/common/src/utils/validateFileName.ts (hunk 1 above)
Comment (aditya-mitra) [inline packages/common/src/utils/validateFileName.ts]: we can restrict the fileName length in the regex in itself. somewhat like `{1,64}`

**Review by aditya-mitra — CHANGES_REQUESTED** [review]

File: packages/common/src/utils/validateFileName.ts (hunk 1 above)
Comment (SYBIOTE) [reply packages/common/src/utils/validateFileName.ts]: Is that necessary, I would not want to bundle the length restriction into the regex, 
We use regex across the codebase for mainly character search, baking it in for a one specific case doesn't feel right.

Take a look at https://tsu.atlassian.net/browse/IR-3169

We can take up advanced regex and make it a part of this PR, the effort will be more centralized

Comment (aditya-mitra) [reply packages/common/src/utils/validateFileName.ts]: We can actually use length restrictions in regex. We have done in the scene name regex - https://github.com/EtherealEngine/etherealengine/blob/0e1e55d8809128472211a42e5d4efc4a66df7324/packages/common/src/regex/index.ts/#L33

Comment (SYBIOTE) [reply packages/common/src/utils/validateFileName.ts]: Gotcha

Comment (SYBIOTE) [reply packages/common/src/utils/validateFileName.ts]: implemented

File: packages/common/tests/regex.test.ts [hunk 2]
```diff
@@ -61,14 +60,23 @@ describe('regex.test', () => {
...
+        'another\ncontrol',
+        'file name', // (a space and an underscore)
+        '< tag >', // (spaces and less-than < and greater-than > characters)
+        'key : value', // (a space and a colon :)
+        'quote " example', // (a space and a double quote ")
+        'path / to / file', // (spaces and forward slashes /)
+        'C:\\ path \\ to \\ file', // (spaces and backslashes \)
+        'pipe | character', // (a space and a pipe |)
+        'question ? mark', // (a space and a question mark ?)
+        'star * char' // (a space and an asterisk *)
```
Comment (aditya-mitra) [inline packages/common/tests/regex.test.ts]: your strings are good and self-explanatory. we should remove comments where they can be understood

**Review by aditya-mitra — DISMISSED** [review]

File: packages/common/tests/regex.test.ts (hunk 2 above)
Comment (SYBIOTE) [reply packages/common/tests/regex.test.ts]: does it matter? ill remove them nonetheless

Comment (SYBIOTE) [reply packages/common/tests/regex.test.ts]: done

**Review by CITIZENDOT — APPROVED** [review]

**Pull request merged**